.github/workflows/build_and_deploy.yml

```diff
@@ -5,8 +5,6 @@ on:
     branches:
       - main
   pull_request:
-    branches:
-      - main
```

Comment:
I've made it run the pipeline on all PRs, not just ones that point at the main branch. This is so we still get the preview URL for all PRs.